src/Microsoft.DotNet.CodeFormatter.Analyzers.Tests/ExplicitVariableTypeAnalyzerTests.cs

```diff
@@ -0,0 +1,399 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.CodeFormatter.Analyzers;
+
+using Xunit;
+
+namespace Microsoft.DotNet.CodeFormatter.Analyzers.Tests
+{
+    public sealed class ExplicitVariableTypeAnalyzerTests : AnalyzerFixerTestBase
```

Comment:
I think something like 'ProvideExplicitVariableType' might be a more complete/descriptive name